ops/travis/e2e-testing/tutorials/sleepers.js

```diff
@@ -0,0 +1,72 @@
+// node sleepers.js [user] [password]
+
+const startBrowser = require('../utils/startBrowser');
+const auto = require('../utils/auto');
+const utils = require('../utils/utils');
+
+const demo = true;
+let {
+  user,
+  pass
+} = utils.getRandUserAndPass();
+
+const args = process.argv.slice(2);
+if (args.length === 2) {
+  user = args[0];
+  pass = args[1];
+}
+
+async function runTutorial (url) {
+  console.log("Running tutorial on", url);
+  const browser = await startBrowser.launch(demo);
+  const page = await browser.newPage();
+  await page.goto(url);
+
+  await auto.register(page, user, pass);
+  // Login
+  await auto.logIn(page, user, pass);
+
+  // Use template to create Sleepers study
+  // Run pipeline
+  const templateName = "Sleepers";
+  await auto.dashboardOpenFirstTemplateAndRun(page, templateName);
+  await page.waitFor(5000);
+
+  // Check results are there
+  await auto.openNode(page, 0);
+  // await auto.openLastNode(page);
+  await page.waitFor(5000);
```

Comment:
I am not sure about these waiting times... could lead to false failings. Wouldn't it be better to wait for changes in the interface and continue when specific UI items appear or reach a certain state?